lib/storage/client.js

```diff
@@ -1044,6 +1044,32 @@ class Client extends CryptoConsumer {
     const params = { id_string: noteName }
     return shared.issueNoteChallenge(this, params, meta)
   }
+
+  /**
+   * createGroup is a static method that creates a groups and sends it to TozStore,
+   * allowing you to supply your own encryptionKey pair.
+   *
+   *
+   * @param {object} data  A hashmap of the data in the group - for now this just includes groupName
```

Comment:
these data objects should get created as types if they are structured

Reply:
ah -- looks like it is based on the shared method below 👍 . Let's call this something more like `groupData` and indicate the type here instead of 'object' in the doc-block.